/**
 * The catch binding in a try â€¦ catch statement would not always be used
 */

try {
} catch (error) {
}

// new
try {
} catch {
}

